api/ShadowRoot.json

```diff
@@ -130,8 +130,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "alternative_name": "cloneable",
-              "version_added": "preview"
+              "version_added": "17.4"
```

Comment:
Should this be the same compat history as Element.clonable?

Reply:
I think in Element.json the situation is slightly different. The `cloneable` name shipped in 16.4 already whereas here it was never shipped outside of `preview`. So I would leave this as is.